src/main/scala/mesosphere/mesos/TaskGroupBuilder.scala

```diff
@@ -157,16 +162,19 @@ object TaskGroupBuilder {
     podDefinition: PodDefinition,
     portsMatch: PortsMatch,
     portMappings: Seq[mesos.NetworkInfo.PortMapping],
-    instanceId: Instance.Id): mesos.ExecutorInfo.Builder = {
+    instanceId: Instance.Id,
+    frameworkId: mesos.FrameworkID): mesos.ExecutorInfo.Builder = {
     // TODO: use only an instance id.
     val executorID = mesos.ExecutorID.newBuilder.setValue(f"marathon-${instanceId.idString}")
 
     val executorInfo = mesos.ExecutorInfo.newBuilder
       .setType(mesos.ExecutorInfo.Type.DEFAULT)
       .setExecutorId(executorID)
+      .setFrameworkId(frameworkId)
 
     executorInfo.addResources(scalarResource("cpus", PodDefinition.DefaultExecutorCpus))
     executorInfo.addResources(scalarResource("mem", PodDefinition.DefaultExecutorMem))
+    executorInfo.addResources(scalarResource("disk", 0))
```

Comment:
So this resource probably shouldn't be hardcoded here, but part of the pod definition...